pkg/client/client.go

```diff
@@ -52,6 +52,8 @@ type KpmClient struct {
 	settings settings.Settings
 	// The flag of whether to check the checksum of the package and update kcl.mod.lock.
 	noSumCheck bool
+	// The package to use in case of multiple packages.
+	pkg string
```

Comment:
It may not be appropriate to add this field to KpmClient, which is the client of the package management tool, and it should not be associated with a specific package.

Reply:
Thanks for the comment @zong-zhe. I have removed it!